packages/shared/src/components/checklist/GoPublicStep.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement } from 'react';
+import { ChecklistStepPropsWithSquad } from '../../lib/checklist';
+import { Button, ButtonVariant } from '../buttons/Button';
+import { ChecklistStep } from './ChecklistStep';
+import { TourIcon } from '../icons';
+import { anchorDefaultRel } from '../../lib/strings';
+import { SquadPostsProgressBar } from '../squads/SquadPostsProgressBar';
+
+export const GoPublicStep = ({
+  squad,
+  ...props
+}: ChecklistStepPropsWithSquad): ReactElement => {
+  const postsCount = (squad?.flags?.totalPosts ?? 0) - 1; // -1 for the welcome post
+
+  return (
+    <ChecklistStep {...props}>
+      <div className="flex flex-col items-stretch gap-4">
+        <SquadPostsProgressBar
+          postsCount={postsCount}
+          className={{
+            progressBackground: 'bg-border-subtlest-tertiary',
+            progressBar: '!bg-border-subtlest-primary',
+            label: 'typo-caption2 *:!font-normal *:!text-text-primary',
```

Comment:
Why do we need all the child selectors here and as important?
It's a niche component can't we fix it down below?

Reply:
The problem is, that we have completely different styling in 2 places that we use this component (different colors, font sizes, text colors).

I will change the code so that we do not use these child selectors 👍